db/db.pl

```diff
@@ -3508,6 +3508,17 @@ sub parseArgs {
       esPost("/${PREFIX}lookups/lookup", to_json($newShortcut));
     }
 
+    # increment the _meta version by 1
+    my $mapping = esGet("/${PREFIX}lookups/_mapping");
+    my @indexes = keys %{$mapping};
+    my $index = @indexes[0];
+    my $meta = $mapping->{$index}->{mappings}->{_meta};
+
+    my $version = $meta->{version};
+    $version++;
+
+    esPut("/${PREFIX}lookups/_mapping", "{\"_meta\":{\"version\": $version, \"initSync\":" . ($meta->{initSync} ? "true" : "false") . "}}");
```

Comment:
Try changing to something like
$meta->{version}++
esPut("/${PREFIX}lookups/_mapping", to_json({"_meta" => $meta}));